posts/2021-08-06-21009-beta.adoc

```diff
@@ -0,0 +1,179 @@
+---
+layout: post
+title: Endpoint activation monitoring and more in Open Liberty 21.0.0.9-beta
+# Do NOT change the categories section
+categories: blog
+author_picture: https://avatars3.githubusercontent.com/ryan-storey
+author_github: https://github.com/ryan-storey
+seo-title: Endpoint activation monitoring and more now available on Open Liberty 21.0.0.9-beta - OpenLiberty.io
+seo-description: Open Liberty 21.0.0.9-beta offers exciting new enhancements for Liberty embedders such as endpoint activation monitoring to determine when endpoints are available to use. 
+blog_description: Open Liberty 21.0.0.9-beta offers exciting new enhancements for Liberty embedders such as endpoint activation monitoring to determine when endpoints are available to use. 
+open-graph-image: https://openliberty.io/img/twitter_card.jpg
+---
+= Endpoint activation monitoring and more now available on Open Liberty 21.0.0.9-beta
+Ryan Storey <https://github.com/ryan-storey>
+:imagesdir: /
+:url-prefix:
+:url-about: /
+//Blank line here is necessary before starting the body of the post.
+
+
+Open Liberty 21.0.0.9-beta offers exciting new enhancements for Liberty embedders such as endpoint activation monitoring to determine when endpoints are available to use. 
+
+We have two beta packages for link:{url-about}[Open Liberty]:
+
+* <<allbeta, All Beta Features>>: a larger package that contains all Open Liberty beta features (including Jakarta EE 9 beta features) and GA features and functions.
+* <<jakarta, Jakarta EE 9 Beta Features>>: a lightweight package that contains only the Jakarta EE 9 features.
+
+This means that you can now try out our in-development Open Liberty features by just adding the relevant coordinates to your build tools.
+
+If you try either package, <<feedback, let us know what you think>>.
+[#allbeta]
+== All Beta Features package
+
+The All Beta Features package includes the following beta features:
+
+* <<method, New method to determine active endpoints>>
+* link:https://openliberty.io/blog/2021/07/12/logging-format-21008-beta.html#mp4_1[MicroProfile 4.1]
```

Comment:
I thought this further. It might be better to create a section on MicroProfile 4.1 and then link to that.
<<mp4.1, MicroProfile 4.1>>

Reply:
Added!